integration/test_fulltext.py

```
@@ -1252,6 +1252,43 @@ def test_mixed_predicates(self):
+        client.execute_command("HSET", "", "content", "backfilltoken")
+        client.execute_command(
+            "FT.CREATE", "idx", "ON", "HASH", "PREFIX", "1", "",
+            "SCHEMA", "content", "TEXT", "NOSTEM"
+        )
+
+        IndexingTestHelper.wait_for_backfill_complete_on_node(client, "idx")
+
+        result = client.execute_command("FT.SEARCH", "idx", "@content:backfilltoken")
+        assert result[0] == 1
+        assert result[1] == b""
+        assert dict(zip(result[2][::2], result[2][1::2])) == {
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)